plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -928,6 +935,13 @@ def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sock
         if searchdomains:
             kwargs['searchdomain'] = ' '.join(searchdomains)
 
+    # VM tags are expected to be valid and presented as a comma/semi-colon delimited string
+    if 'tags' in module.params:
```

Comment:
`'tags'` is always present, but it can be `None` if the user did not specify it:
```suggestion
    if module.params['tags']:
```

Reply:
Thank you for catching this.